pkg/handlers/config.go

```diff
@@ -100,32 +100,21 @@ func UpdateAppConfig(w http.ResponseWriter, r *http.Request) {
 	requiredItemsTitles := make([]string, 0, 0)
 	for _, group := range updateAppConfigRequest.ConfigGroups {
 		for _, item := range group.Items {
-			if !item.Required {
-				continue
-			}
-			if item.Hidden || item.When == "false" {
-				continue
-			}
-			if !(item.Value.Type == multitype.String && item.Value.String() == "") {
-				continue
-			}
-			if !(item.Default.Type == multitype.String && item.Default.String() == "") {
-				continue
-			}
-			requiredItems = append(requiredItems, item.Name)
-			if item.Title != "" {
-				requiredItemsTitles = append(requiredItemsTitles, item.Title)
-			} else {
-				requiredItemsTitles = append(requiredItemsTitles, item.Name)
+			if app.IsUnsetRequiredItem(&item) {
```

Comment:
This is overloaded a bit.  What if we split the checks for IsRequiredItem and IsUnset into two methods?